src/components/ui/CaptionInput.js

```diff
@@ -0,0 +1,41 @@
+import { X as X } from '@styled-icons/boxicons-regular'
+import styled from 'styled-components/macro'
+
+import CircleIcon from './CircleIcon'
+import IconButton from './IconButton'
+import Input from './Input'
+import ListEntry from './ListEntry'
+
+const StyledCaptionInput = styled(ListEntry)`
+  padding: 8px;
+  height: 16px;
+  display: flex;
```

Comment:
Your height seems off, since the rest of your elements have height 36px.

I don't think you need display: flex.

Reply:
I also don't think you need padding here. If you want space between each CaptionInput, it's best to add a `margin-bottom: 16px`.